Content.Server/Ghost/Roles/Components/TakeGhostRoleEvent.cs

```diff
@@ -3,7 +3,7 @@
 namespace Content.Server.Ghost.Roles.Components;
 
 [ByRefEvent]
-public record struct TakeGhostRoleEvent(IPlayerSession Player)
+public record struct TakeGhostRoleEvent(IPlayerSession Player, string UserId)
```

Comment:
`IPlayerSession` already has a `UserId` field (the netuserid)? is this necessary?

Reply:
I might have missed `.netuserid` as a possibility, I'll need to review.